ironfish/src/node.ts

```diff
@@ -139,6 +139,10 @@ export class IronfishNode {
       this.telemetry.submitBlockMined(block)
     })
 
+    this.peerNetwork.onTransactionReceived.on((transaction, date) => {
+      this.telemetry.submitNewTransactionSeen(transaction, date)
```

Comment:
this is a small nit but in the call site you call this `received` which makes it clear what this is, but here it's a date. It's more clear at the call site that this is the received at date. I like you original name more.

Reply:
agreed